stable/gangway/README.md

```diff
@@ -55,6 +55,7 @@ At a minimum you *must* configure any of the values marked as **required** in th
 | `gangway.tokenURL`                | OAuth2 URL to obtain access tokens. To be taken from the configuration of your OIDC provider. **Required**                                                                                                      | `""`                                               |
 | `gangway.trustedCAPath`           | The path to a root CA to trust for self signed certificates at the Oauth2 URLs                                                                                                                                  | `""`                                               |
 | `gangway.usernameClaim`           | The JWT claim to use as the username. This is used in UI. Default is "nickname". This is combined with the clusterName for the "user" portion of the kubeconfig.                                                | `name`                                             |
+| `trustedCAEnabled`                | Should Gangway use a root CA to trust for self signed certificates at the Oauth2 URLs?  If true, you must create a file rootca.crt in this directory containing the certificate.                                | `false`                                            |
```

Comment:
If you accept the proposal to change `configmap.yaml` this will need changing slightly.

Reply:
Pls see comment above.